### ISSUE TITLE
[Feature request] Switch between open windows within the active workspace

### ISSUE THREAD

**Issue by marekciupak** [pop-os/cosmic-settings#650]
Hi everyone! :wave: 

Currently the settings support setting any shortcuts for switching between open windows (feature commonly known as "Alt+Tab"):

```
Input Devices -> Keyboard -> Keyboard Shortcuts -> System -> Switch between open windows
```

I would like to request for an additional shortcut that would work exactly the same way, but would only include windows from the active workspace.

I think this will be very useful because it will allow you to dedicate specific workspaces to specific contexts (work, entertainment, etc.) and keep switching between windows only in the selected context.

Please consider my request. Thank you.

**Comment by Toniolo-Marco** [comment]
It should be in Navigation section, I guess.

**Comment by amzker** [comment]
any updates on this? or like any way we can manually configure it ?

**Comment by RichardJActon** [comment]
Yeah this would be a great addition - I've just been trying out cosmic and the inability to do this threw me. I like my window switching separated by workspace.

**Comment by brunizzl** [comment]
Just another reason this might be interesting: In windows with standard settings, programs which currently run in other workspaces show up nowhere on your screen, including alt-tab and the taskbar. If ex-windows users are part of the target audience, one should at least add a setting to enable the established workflow for these people (me being one of them).

**Comment by EVV1E** [comment]
I agree with all comments above. This is one of the biggest pain points I have with COSMIC right now. I frequently use Alt+Tab for cycling between floating windows like I'm used to from other DEs and operating systems but this gets really confusing and annoying when windows that aren't in the current workspace are targeted so often
